templates/resources/fleet_package.md.tmpl

```diff
@@ -0,0 +1,20 @@
+---
+subcategory: "Fleet"
+layout: ""
+page_title: "Elasticstack: elasticstack_fleet_package Resource"
+description: |-
+  Installs or uninstalls a Fleet integration package.
+---
+
+# Resource: elasticstack_fleet_package
+
+Installs or uninstalls a Fleet integration package.
```

Comment:
```suggestion
Installs or uninstalls a Fleet integration package. The Kibana Fleet UI can be used to view available packages. 
```

It's probably worth linking through to the [docs](https://www.elastic.co/guide/en/fleet/8.10/install-uninstall-integration-assets.html) from here as well, even though they just point back to Kibana.